WalletWasabi.Fluent/WalletWasabi.Fluent.csproj

```diff
@@ -37,4 +37,16 @@
 	<ItemGroup>
 		<RuntimeHostConfigurationOption Include="System.Globalization.Invariant" Value="true" />
 	</ItemGroup>
+
+	<ItemGroup>
+	  <Compile Update="Views\CoinSelection\Core\Cells\CoinSelectionCellView.axaml.cs">
+	    <DependentUpon>CoinSelectionCellView.axaml</DependentUpon>
+	  </Compile>
+	  <Compile Update="Views\CoinSelection\Core\Cells\SelectableCollectionCellView.axaml.cs">
+	    <DependentUpon>SelectableCollectionCellView.axaml</DependentUpon>
+	  </Compile>
+	  <Compile Update="Views\CoinSelection\Core\Headers\SelectionHeaderView.axaml.cs">
+	    <DependentUpon>SelectionHeaderView.axaml</DependentUpon>
+	  </Compile>
+	</ItemGroup>
```

Comment:
Seems like this is redundant? it is not required for other views.

Reply:
Yes, it is. Visual Studio seems to be really bad handling file addition. It sometimes adds entries to the csproj unadvertedly 😣.